scripts/create-common-bundle/index.js

```diff
@@ -5,6 +5,10 @@ import glob from 'tiny-glob/sync.js';
 import { fileURLToPath } from 'url';
 import { execSync } from 'child_process';
 
+if (!!process.env.VERCEL) {
+	execSync('git clean -d -f content/tutorial');
+}
```

Comment:
When moving or editing exercises locally, we don't want to accidentally remove files. For this reason, this should only be run on Vercel.